apps/mempool/docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+version: "3.7"
+
+x-logging:
+  &default-logging
+  driver: journald
+  options:
+    tag: "umbrel-app {{.Name}}"
+
+services:
+  app:
+    image: bguillaumat/mempool-frontend:v2.0.1
+    logging: *default-logging
+    restart: on-failure
+    stop_grace_period: 1m
+    ports:
+      - ${APP_MEMPOOL_PORT}:80
+    links:
+      - api
+    environment:
+      API_IP: $APP_MEMPOOL_API_IP
+    networks:
+      default:
+        ipv4_address: $APP_MEMPOOL_IP
+  api:
+    image: bguillaumat/mempool-backend:v2.0.1
+    logging: *default-logging
+    restart: on-failure
+    stop_grace_period: 1m
+    command: "./wait-for-it.sh db:3306 --timeout=720 --strict -- ./start.sh"
+    environment:
+      RPC_HOST: $BITCOIN_IP
+      RPC_PORT: $BITCOIN_RPC_PORT
+      RPC_USER: $BITCOIN_RPC_USER
+      RPC_PASS: $BITCOIN_RPC_PASS
+      ELECTRS_HOST: $ELECTRUM_IP
+      ELECTRS_PORT: $ELECTRUM_PORT
+      MYSQL_HOST: $APP_MEMPOOL_DB_IP
+    expose:
+      - "8999"
+    links:
+      - db
+    networks:
+       default:
+         ipv4_address: $APP_MEMPOOL_API_IP
+  db:
+    image: bguillaumat/mempool-db:v2.0.1
+    logging: *default-logging
+    restart: on-failure
+    stop_grace_period: 1m
+    volumes:
+      - ${APP_DATA_DIR}/mysql:/var/lib/mysql
+    environment:
+      MYSQL_USER: "mempool"
+      MYSQL_PASSWORD: "mempool"
+    expose:
+     - "3306"
+    networks:
+       default:
+         ipv4_address: $APP_MEMPOOL_DB_IP
```

Comment:
It may be better to directly use official MariaDB/mysql images here (see [Luke's review](https://github.com/getumbrel/umbrel/pull/461#discussion_r566724170) and b444272 as I did exactly this today).

Reply:
Hum I guess it can be better but since my other repo split every files and format them so we can easily track mempool updates it's will be a "lost of time" to copy the files here every time we update the app.
If you really think we needs to put it here I will :smiley:

Reply:
Yea, thinking about this more, it's fine if you guys wanna do it like this.

I'm thinking ahead about how app developers will want to package apps when the Umbrel app configuration lives in the app's own repo. It makes sense in that case because they will probably already have an existing Docker image, so they don't want to create and maintain yet another child Dockerfile specifically for Umbrel, it's handy if they can use oll their existing Docker images (or official images) and then just pass in some Umbrel specific init scripts.

But that's not how things work right now, and you guys are already having to create your own Docker images for many projects, so I guess there's no issue with you including whatever you want directly in the image. It also means you can edit and maintain them all in one place.

So it's fine to keep doing it like this. (sorry Lounes! feel free to undo your changes if you want.)